Library/Homebrew/cmd/audit.rb

```diff
@@ -363,10 +363,16 @@ def audit_deps
           problem "Don't use git as a dependency"
         when "mercurial"
           problem "Use `depends_on :hg` instead of `depends_on 'mercurial'`"
-        when "ruby"
-          problem "Don't use ruby as a dependency. We allow non-Homebrew ruby installations."
         when "gfortran"
           problem "Use `depends_on :fortran` instead of `depends_on 'gfortran'`"
+        when "ruby"
```

Comment:
And completely irrational, if you'd ask me. (Or just an argument that doesn't work given how my brain is wired.) :wink: But I can deal with that.